robotpageobjects/page.py

```diff
@@ -154,7 +155,13 @@ def __init__(self):
 
         self._attempt_sauce = self._validate_sauce_options()
 
-        # There's only a session ID when using a remote webdriver (Sauce, for example)
+        self._Capabilities = getattr(webdriver.DesiredCapabilities, self.browser.upper())
+        for grid_cap in self._Capabilities:
```

Comment:
Seems like we're checking all options in `DesiredCapabilities` here not just grid-related ones.  I think that's fine, but  we shouldn't use "grid" in these variable names.